docs/cli.md

```diff
@@ -81,6 +81,14 @@ You can pass in format options with `--format-options <JSON>`. The JSON string m
 
 * Suggested use: add with profiles so you can define an object and use `JSON.stringify` instead of writing `JSON` manually.
 
+## ES Modules (experimental)
+
+You can optionally write your support code (steps, hooks, etc) with native ES modules syntax - i.e. using `import` and `export` statements without transpiling.
+
+To enable this, run with the `--esm` flag.
+
+This will also expand the default glob for support files to include the `.mjs` file extension.
```

Comment:
Is ESM purely additive (ie require is still supported)?

If not, would seem like using this requires custom formatters used would also need to be written in ESM

Reply:
ESM is additive. If enabled, it'll use `import()` to load all user code (steps/hooks, formatters, snippets, and config file). If any of this user is CommonJS and has `require()`s in it, that will still work fine - [you can `import()` CommonJS modules, but you can't `require()` ESM](https://nodejs.org/api/esm.html#esm_interoperability_with_commonjs).

(Hope I've understood the question right!)

Reply:
And I'll push some documentation improvements to this effect.

Reply:
Looks good. Thanks!